scripts/run.sh

```diff
@@ -23,7 +23,7 @@ download() {
 
     source ~/gnu-koryavov/config.conf
 
-    cd ~/gnu-koryavov/KORYAVNIKS/ && wget -O ${sem}.djvu ${KORYAVNIKS[$sem]}
+    cd ~/gnu-koryavov/KORYAVNIKS/ && wget -q --show-progress -O ${sem}.djvu ${KORYAVNIKS[$sem]}
```

Comment:
Maybe redirect `stdout` to `/dev/null` as we only need to check for errors output? All normal output should be supressed,

Reply:
Suppressing all the output leads to the lag that can make end user think that `gnu-koryavov` is stuck (it's especially evident with a slow internet connection)